lib.py

```diff
@@ -3,9 +3,11 @@
 import json, random
 from scipy.stats import truncnorm
 
+JSON_FILE_SUFFIX = 'trait_tree.json'
```

Comment:
Concerning this change in particular, I am actually shorty pushing up a change to the way we do these suffixes having each trait_tree be unique to each generated person with a different unique suffix for the file name, so we might want to drop the changes to lib until that gets in.